src/server/config.rs

```diff
@@ -213,11 +213,19 @@ impl Config {
                     ));
                 }
             } else {
-                info!(
-                    "no advertise-status-addr is specified, falling back to status-addr";
-                    "status-addr" => %self.status_addr
-                );
-                self.advertise_status_addr = self.status_addr.clone();
+                if !self.status_addr.starts_with("0.0.0.0") {
```

Comment:
What about parsing the address and use `is_unspecified` to check if it is legit?

Reply:
We had validated the address by `config::check_addr`

Reply:
what if it's "::/0"?

Reply:
> What about parsing the address and use `is_unspecified` to check if it is legit?

Done, PTAL again